### PR TITLE
Fix/add protein ids

### PR DESCRIPTION
**Pull request by timosachsenberg**

### CODE REVIEW

**Comment by timosachsenberg** [comment]
minor fix to add back the protein id to the (kind of deprecated) tsv output

File: src/openms/source/ANALYSIS/RNPXL/RNPxlReport.cpp [hunk 1]
```diff
@@ -107,7 +107,7 @@ namespace OpenMS
         sl << String(ma.first + "_" + ma.second[i].first);
       }
     }
-    sl << "abs prec. error Da" << "rel. prec. error ppm" << "M+H" << "M+2H" << "M+3H" << "M+4H" << "fragment_annotation" << "rank";
+    sl << "abs prec. error Da" << "rel. prec. error ppm" << "M+H" << "M+2H" << "M+3H" << "M+4H" << "rank";
```
Comment (enetz) [inline src/openms/source/ANALYSIS/RNPXL/RNPxlReport.cpp]: @timosachsenberg wasn't this in another PR, that you closed without merging recently? (Just to make sure this was on purpose.) Also, are you just removing the column header without changing the actual output in the following rows, or did you do that somewhere else? Because this seems unrelated to the other change in this PR.

**Comment by timosachsenberg** [comment]
replaced by #3720
